pippy/LoadModule.py

```diff
@@ -20,7 +20,22 @@ def load_checkpoint(
     device: torch.device = None,
     dtype: torch.dtype = None,
     checkpoint_prefix: str = None,
-):
+) -> nn.Module:
+    """
+    Load a checkpoint from a model file.
+    Args:
+        model: the model to load the checkpoint into
```

Comment:
nit: for consistency, let's also include the variable type in args as you did for _get_file_to_weight_map()?